src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -1244,12 +1275,56 @@ namespace AirflowNetworkBalanceManager {
-						} else if ( !SameString(Alphas( 4 ), "Absolute" ) ) {
+						} else if ( !SameString( Alphas( 4 ), "Absolute" ) ) {
 							ShowWarningError( RoutineName + CurrentModuleObject + " object, Invalid input " + cAlphaFields( 4 ) + " = " + Alphas( 4 ) );
 							ShowContinueError( "The default value is assigned as Absolute." );
 						}
 					}
 				}
+ 				CurrentModuleObject = "OutdoorAir:Node";
+ 				for ( i = AirflowNetworkNumOfExtNode - AirflowNetworkNumOfOutAirNode + 1; i <= AirflowNetworkNumOfExtNode; ++i ) {
+ 					GetObjectItem( CurrentModuleObject, i - ( AirflowNetworkNumOfExtNode - AirflowNetworkNumOfOutAirNode ), Alphas, NumAlphas, Numbers, NumNumbers, IOStatus, lNumericBlanks, lAlphaBlanks, cAlphaFields, cNumericFields );
+ 					IsNotOK = false;
+ 					IsBlank = false;
+ 					VerifyName( Alphas( 1 ), MultizoneExternalNodeData, i - 1, IsNotOK, IsBlank, CurrentModuleObject + " Name" );
+ 					if ( IsNotOK ) {
+ 						ErrorsFound = true;
+ 						if ( IsBlank ) Alphas( 1 ) = "xxxxx";
+ 					}
+						
+					if ( NumAlphas > 5 && !lAlphaBlanks( 6 ) ) { // Wind pressure curve
+						MultizoneExternalNodeData( i ).curve = GetCurveIndex( Alphas( 6 ) );
+						if ( MultizoneExternalNodeData( i ).curve == 0 ) {
+							ShowSevereError( RoutineName + "Invalid " + cAlphaFields( 6 ) + "=" + Alphas( 6 ) );
+							ShowContinueError( "Entered in " + CurrentModuleObject + '=' + Alphas( 1 ) );
+							ErrorsFound = true;
+						}
+					}
+
+					if ( NumAlphas > 6 && !lAlphaBlanks( 7 ) ) { // Symmetric curve
+						if ( SameString( Alphas( 7 ), "Yes" ) ) {
+							MultizoneExternalNodeData( i ).symmetricCurve = true;
+						} else if ( !SameString(Alphas( 7 ), "No" ) ) {
+							ShowWarningError( RoutineName + CurrentModuleObject + " object, Invalid input " + cAlphaFields( 7 ) + " = " + Alphas( 7 ) );
+							ShowContinueError( "The default value is assigned as No." );
+						}
+					}
+
+					if ( NumAlphas > 7 && !lAlphaBlanks( 8 ) ) { // Relative or absolute wind angle
+						if ( SameString( Alphas( 8 ), "Relative" ) ) {
+							MultizoneExternalNodeData( i ).useRelativeAngle = true;
+						} else if ( !SameString(Alphas( 8 ), "Absolute" ) ) {
+							ShowWarningError( RoutineName + CurrentModuleObject + " object, Invalid input " + cAlphaFields( 8 ) + " = " + Alphas( 8 ) );
+							ShowContinueError( "The default value is assigned as Absolute." );
+						}
+					}
+
+ 					MultizoneExternalNodeData( i ).Name = Alphas( 1 ) ; // Name of external node
+ 					NodeNum = GetOnlySingleNode( Alphas( 1 ) , ErrorsFound, CurrentModuleObject, "AirflowNetwork:Multizone:Surface", NodeType_Air, NodeConnectionType_Inlet, 1, ObjectIsParent );
+ 					MultizoneExternalNodeData( i ).OutAirNodeNum = NodeNum; // Name of outdoor air node
+ 					MultizoneExternalNodeData( i ).height = Node( NodeNum ).Height; // Nodal height
+ 					MultizoneExternalNodeData( i ).ExtNum = AirflowNetworkNumOfZones + i; // External node number
```

Comment:
This looks wrong: ExtNum = AirflowNetworkNumOfZone + i. How does external node number relate to number of zones?